### PR TITLE
fix: MD003/heading-style/header-style

### PR DESCRIPTION
**Pull request by nschonni**
- Suppress the H2 versions since I believe their is a TOC bug.
- Escape trailing hashmark in title
- Trim trailing/leading space
- MD020 also overlaps the same lines

### CODE REVIEW

**Comment by PRMerger18** [comment]
@nschonni : Thanks for your contribution! The author, @John-Hart, has been notified to review your proposed change.

**Comment by John-Hart** [comment]
Most of this is outside the VSTO docs that I own.  @gewarren could you help or help find the right people to review this?

File: docs/test/how-to-create-a-load-test-plug-in.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ You can create a load test plug-in to run code at different times while the load
 
 [!INCLUDE [web-load-test-deprecated](includes/web-load-test-deprecated.md)]
 
+<!-- markdownlint-disable MD003 MD020 -->
 ## To create a load test plug-in in C#
+<!-- markdownlint-disable MD003 MD020 -->
```
Comment (gewarren) [inline docs/test/how-to-create-a-load-test-plug-in.md]: enable

Comment (nschonni) [reply docs/test/how-to-create-a-load-test-plug-in.md]: thanks! fixed

**Comment by PRMerger8** [comment]
@nschonni : Thanks for your contribution! The author, @John-Hart, has been notified to review your proposed change.

**Pull request merged**